packages/frontend/src/components/molecules/DashboardCard.tsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
+import './DashboardCard.scss';
 
 interface IProps {
-  title: string;
+  value: number;
```

Comment:
I don't think we're always going to have a number there ! We could have for example a number with a percentage so i think that string is more adapted here.